CHANGELOG.md

```diff
@@ -2,10 +2,349 @@
 
 ## [2.4.0](https://launchpad.net/mixxx/+milestone/2.4.0) (Unreleased)
 
-* Cover art: Prevent wrong cover art display due to hash conflicts
-* Cover art: Add background color for quick cover art preview
+### Cover Art ###
+* Prevent wrong cover art display due to hash conflicts [#2524](https://github.com/mixxxdj/mixxx/pull/2524)
+* Add background color for quick cover art preview [#2524](https://github.com/mixxxdj/mixxx/pull/2524)
+
+### Music Library ###
+* Ensure that tracks with an invalid bpm are re-analyzed [#2776](https://github.com/mixxxdj/mixxx/pull/2776)
+* Engine Prime export using djinterop
+  [#2753](https://github.com/mixxxdj/mixxx/pull/2753)
+  [#2932](https://github.com/mixxxdj/mixxx/pull/2932)
+  [#3102](https://github.com/mixxxdj/mixxx/pull/3102)
+  [#3155](https://github.com/mixxxdj/mixxx/pull/3155)
+  [#3621](https://github.com/mixxxdj/mixxx/pull/3621)
+  [#3776](https://github.com/mixxxdj/mixxx/pull/3776)
+  [#3787](https://github.com/mixxxdj/mixxx/pull/3787)
+  [#3797](https://github.com/mixxxdj/mixxx/pull/3797)
+  [#3798](https://github.com/mixxxdj/mixxx/pull/3798)
+  [#4025](https://github.com/mixxxdj/mixxx/pull/4025)
+  [#4087](https://github.com/mixxxdj/mixxx/pull/4087)
+  [#4102](https://github.com/mixxxdj/mixxx/pull/4102)
+  [#4143](https://github.com/mixxxdj/mixxx/pull/4143)
+* Rekordbox: Save all loops and correct AAC timing offset for CoreAudio [#2779](https://github.com/mixxxdj/mixxx/pull/2779)
+* Improve messages during schema migration [#2979](https://github.com/mixxxdj/mixxx/pull/2979)
+* Fix "[Library],GoToItem" in tracks table [#3196](https://github.com/mixxxdj/mixxx/pull/3196)
```

Comment:
Is this broken in 2.3? Or is this a regression fix? In that case it should not be part of the changelog.

Reply:
This has been backported to 2.3.0 .. removed.